src/jvm/main/org/jetbrains/kotlinx/lincheck/paramgen/ExpandingRangeIntGenerator.java

```diff
@@ -64,12 +74,25 @@ public class ExpandingRangeIntGenerator {
      */
     public ExpandingRangeIntGenerator(Random random, int begin, int end, int minBegin, int maxEnd) {
         this.random = random;
+
+        this.initialBegin = begin;
```

Comment:
begin -> start. Please also add incluside/exclusive suffixes to make the semantics clearer